PowerPointLabs/PowerPointLabs/Utils/ShapeUtil.cs

```diff
@@ -984,6 +984,28 @@ public static TextRange ConvertTextRange2ToTextRange(TextRange2 textRange2)
             return shape == null ? null : shape.TextFrame.TextRange;
         }
 
+        /// <summary>
+        /// Useful when shape has text with more than one color
+        /// Getting color in that case without a workabout will return black
+        /// Assumes that shape has a TextRange
+        /// 
+        /// Unsuccessful workabouts:
```

Comment:
Not sure if the unsuccessful workabouts should be mentioned in summary or not (put as comments?) since it makes the summary very long and as you hover on the method, it extends too long across the screen.

Reply:
Good point. I'll shift it into the function